subprojects/build-cache-http/src/integTest/groovy/org/gradle/caching/http/internal/HttpBuildCacheServiceErrorHandlingIntegrationTest.groovy

```diff
@@ -62,7 +62,8 @@ class HttpBuildCacheServiceErrorHandlingIntegrationTest extends AbstractIntegrat
         withBuildCache().succeeds "customTask"
 
         then:
-        output ==~ /(?s).*org\.gradle\.caching\.BuildCacheException: Unable to store entry at .*: ${errorPattern}.*/
+        output =~ /Could not store entry .* for task :customTask in remote build cache/
```

Comment:
This test fails currently. We need to use the display name in the message, not `toString()`. The message is generated here: https://github.com/gradle/gradle/blob/941559e020f6c357ebb08d5c67acdb858a3defc2/subprojects/core/src/main/java/org/gradle/caching/internal/controller/service/BaseBuildCacheServiceHandle.java#L99